clients/java/src/main/java/io/camunda/client/api/command/CompleteJobCommandStep1.java

```diff
@@ -89,5 +90,74 @@ interface CompleteJobCommandStep2 extends FinalCommandStep<CompleteJobResponse>
      * @return the builder for this command.
      */
     CompleteJobCommandStep2 denied(boolean denied);
+
+    /**
+     * Attributes that were corrected by the worker. The below attributes can be corrected and
+     * additional attributes will be ignored: `assignee` - reset by providing an empty String
+     * `dueDate` - reset by providing an empty String `followUpDate` - reset by providing an empty
+     * String `candidateGroups` - reset by providing an empty list `candidateUsers` - reset by
```

Comment:
> reset by

🔧💭 As Nico pointed out [here](https://github.com/camunda/camunda/pull/25654#discussion_r1880400688), using "reset" might be misleading for users.
We can use "clear" instead, which seems to sound also fine, e.g `clear by providing an empty String`.